front-thor.php

```diff
@@ -32,6 +31,21 @@
 
 namespace Required\FrontThor;
 
+/**
+ * Download language packs from Traduttore.
+ *
+ * @since 1.0.3
+ */
+function init_traduttore() {
+	Required\Traduttore_Registry\add_project(
+		'plugin',
+		'front-thor',
+		'https://translate.required.com/api/translations/required/front-thor/'
+	);
+}
+
```

Comment:
The current style in this plugin is not to have a space between the function end and the `add_action()`.

Reply:
That's not a show stopper though is it? 😉